### ISSUE TITLE
Model creation

### ISSUE THREAD

**Issue by elboyran** [dianna-ai/dianna#392]
- **Model creation**

- [x] Dataset (Train and validation)
- [x] Training
- [x] Validation accuracy

**Comment by elboyran** [comment]
To update the respective section from the [README (in my repo and proper branch)](https://github.com/elboyran/models/tree/leafsnap30-model/vision/classification/leafsnap30) for our model submission.

**Comment by loostrum** [comment]
README is updated, see PR in Elena's repo: https://github.com/elboyran/models/pull/7
I've also opened a PR in dianna-exploration to add the correct hyperparameters to the training notebook (https://github.com/dianna-ai/dianna-exploration/pull/154).
